libs/ui/elements/src/lib/metadata-info/metadata-info.component.html

```diff
@@ -4,11 +4,12 @@
 >
   record.metadata.about
 </p>
-<div class="mb-6 md-description sm:mb-4 sm:pr-16" *ngIf="metadata.abstract">
+<div class="text-gray-900 mb-6 md-description sm:mb-4 sm:pr-16">
```

Comment:
The text should be the `main` text color, I think it means that no color class is required.
We should not use `text-gray` things I guess.

Reply:
> The text should be the `main` text color, I think it means that no color class is required. We should not use `text-gray` things I guess.

the gray-900 is derived from the main color, so that should not be a problem. the main color is almost black and here we wanted a lighter color (as you also mentioned in your review).
cc @tkohr

Reply:
Ok, I thought using the `main` color would allow more consistence with other components, but looking at the card on the home page we also use the derived `text-gray-900`, so it seems ok to me, as long as we try to stay consistent with the derived grays ;-)

Reply:
Why do you remove `*ngIf="metadata.abstract"` ? Is it the abstract container or not ?

And yes, the abstract should have the `main` color, and not `text-gray-900` I guess.

I think the colors are:
- Abstract title: **black**
Abstract content: **main**

- D’où viennent ces données ? title: **black**
Text underneath: **main**
Property label: **grayer**
Property color: **primary**

Reply:
The ngIf was not removed but just moved to the paragraph. Otherwise the content ghost will not be shown.

Regarding the colors:
main is almost black, so compared to the mockups it would be too dark. 
![image](https://github.com/geonetwork/geonetwork-ui/assets/133115263/179bfd76-71b3-4562-96ce-2dbb2d02d87b)

Maybe we shouldn't use pure `black` but `main` because [pure black is apparently bad practice](https://uxplanet.org/basicdesign-never-use-pure-black-in-typography-36138a3327a6) and it's not dependent on the theme.